eth/gasprice/gasprice_test.go

```diff
@@ -109,7 +109,7 @@ func newTestBackendFakerEngine(t *testing.T, config *params.ChainConfig, numBloc
 	engine := dummy.NewETHFaker()
 
 	// Generate testing blocks
-	_, blocks, _, err := core.GenerateChainWithGenesis(gspec, engine, numBlocks, 0, genBlocks)
+	_, blocks, _, err := core.GenerateChainWithGenesis(gspec, engine, numBlocks, config.FeeConfig.TargetBlockRate-1, genBlocks)
```

Comment:
just in case this matters, this changed from `0` to `config.FeeConfig.TargetBlockRate-1`=`1`